staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -1152,7 +1153,7 @@ func (e *Store) DeleteCollection(ctx context.Context, deleteValidation rest.Vali
 					return
 				}
 				if _, _, err := e.Delete(ctx, accessor.GetName(), deleteValidation, options); err != nil && !apierrors.IsNotFound(err) {
-					klog.V(4).Infof("Delete %s in DeleteCollection failed: %v", accessor.GetName(), err)
+					klog.V(4).InfoS("Delete accessor in DeleteCollection failed", "accessor", accessor.GetName(), "err", err)
```

Comment:
s/accessor/object

can you use `klog.KObj`(accessor)? (note that the type of `accessor` is `Object`)

Reply:
updated the code with klog.KObj(accessor) already